src/vs/nls.build.js

```diff
@@ -169,7 +169,6 @@ var NLSBuildLoaderPlugin;
                 bundles: this._getEntryPointsMap()
             }, null, '\t'));
         };
-        ;
```

Comment:
I'm not even sure this file is used anymore - VS Code got rid of the compiled versions a while back and so I assume we also rebuild this. But that's not something I have time to look into now so just fixing the small issue here and leaving it as is otherwise.